style.css

```diff
@@ -143,6 +149,11 @@ section {
   flex-grow: 0;
 }
 
+.img1 img {
+  width: 295px;
+  height: 220px;
+}
+
 .tonic {
```

Comment:
- [ ] Your works header is overlapping the text below it. Kindly fix that.
![image](https://user-images.githubusercontent.com/61919617/172725570-23c9fdf8-42d6-4b1d-bb87-63b68c599569.png)